library/std/src/process/tests.rs

```diff
@@ -96,9 +96,23 @@ fn stdout_works() {
 #[test]
 #[cfg_attr(any(windows, target_os = "vxworks"), ignore)]
 fn set_current_dir_works() {
+    // On many Unix platforms this will use the posix_spawn path.
```

Comment:
This test is `any(windows, target_os = "vxworks")`, is it right place to test musl :-)?